packages/react-instantsearch-hooks-web/src/ui/__tests__/RefinementList.test.tsx

```diff
@@ -645,4 +650,20 @@ describe('RefinementList', () => {
       'Some custom title'
     );
   });
+
+  test('renders with translations', () => {
+    const props = createProps({
+      showMore: true,
+      translations: {
+        showMoreButtonText({ isShowingMore }) {
+          return isShowingMore ? 'Show less brands' : 'Show more brands';
+        },
+      },
+    });
+    const { getByRole } = render(<RefinementList {...props} />);
+
+    expect(
+      getByRole('button', { name: 'Show more brands' })
+    ).toBeInTheDocument();
+  });
```

Comment:
Should we test the translations for the SFFV parts?

Reply:
It's not part of the UI component, this one just has translations for the "show more" button, it's the widget that handles searchbox and no results